### ISSUE TITLE
Experimental keys on scalar is now forbidden

### ISSUE THREAD

**Issue by eserte** [sojiro14/JSON-MergePatch#2]
With perl 5.23.1 the tests fail with

```
#     Error:  Experimental keys on scalar is now forbidden at /tmpfs/.cpan-build/2015072021/JSON-MergePatch-0.01-MwYi6T/blib/lib/JSON/MergePatch.pm line 26.
```

This is probably also the reason why the test suite fails for older perls, see
http://matrix.cpantesters.org/?dist=JSON-MergePatch%200.01

**Comment by sojiro14** [comment]
Thanks for your advice.
I'll try to fix it.

**Comment by eserte** [comment]
It fails also with perl 5.23.2.

**Comment by sojiro14** [comment]
This problem was fixed on version 0.02.
https://github.com/sojiro14/JSON-MergePatch/commit/af36f93de56b3a012520e6c0c4a592c23de45d6b
